### ISSUE TITLE
[Bug]: Cannot leave a space when I am a non-admin and the space has no nodes

### ISSUE THREAD

**Issue by shyamvalsan** [netdata/netdata-cloud#962]
### Bug description

If I am a non-admin in a space then I can't leave the space since the space settings are disabled

![image](https://github.com/netdata/netdata-cloud/assets/24860547/4090363e-6d39-4f27-927b-73f71cced3c8)

### Expected behavior

The user should always have access to the space settings that allow them to leave the space.

### Steps to reproduce

1. Create a space, don't add nodes, invite a non-admin

### Screenshots

See above

### Desktop

Browser Version [e.g. 22]

### Additional context

Now if there are active nodes, then there is a workaround, by clicking on room settings user can reach the leave space button. But even this is not ideal. 

If there are no nodes then the user is stuck in the space forever.

**Comment by hugovalente-pm** [comment]
if there is the permission to leave the space (as well as other space info) the cog needs to be unblocked
https://learn.netdata.cloud/docs/configuring/securing-netdata-agents/role-based-access-model#space-management

**Comment by hugovalente-pm** [comment]
this is related to this https://github.com/netdata/cloud-frontend/issues/4316

**Comment by hugovalente-pm** [comment]
duplicate of the above
